index.html

```diff
@@ -9,8 +9,32 @@
 		<link rel="stylesheet" href="styles.css">
 	</head>
 
-	<body>
+	<body class="body">
+		<header>
+			<div class="header">
+					<img src="image_geometry_1.svg" alt="geometric image" class="image_header">
+					<img src="ic_menu.svg" alt="menu" class="ic_image">
+			</div>
+		</header>
 
+		<div class="headline">
+			<div class="name">
+				Akumu
+				Bavon
+			</div>
+
+			<div class="intro">
+				Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+				If you like what you see and have a project you need coded, don’t hesitate and contact me.
+			</div>
+			<div class="colaboration_button">
+				<button class="colaboration_button_rectangle">
+					Start collaboration
+				</button>
+			</div>
+			<img src="image_geometry_2.svg" alt="geometric image" class="bottom_geometry">
+		</div>
```

Comment:
Hi @Bavon101 ✋. The container of the headline section should be a section tag. Please use the `section` tag instead of `div` for the headline class. You can check [this article](https://www.freecodecamp.org/news/semantic-html5-elements/) for more information about semantic tags.